fairlearn/metrics/_plotter.py

```diff
@@ -0,0 +1,289 @@
+
+
+def plot_metric_frame(metric_frame: MetricFrame,
+                      kind: str = "scatter",
+                      metrics: Union[List[str], str] = None,
+                      error_bars: Union[List[str], str] = None,
+                      conf_intervals: Union[List[str], str] = None,
+                      axs=None,
+                      show_plot=True,
+                      plot_error_labels: bool = False,
+                      text_precision_digits: int = 4,
+                      text_fontsize: int = 8,
+                      text_color: str = "black",
+                      text_ha: str = "center",
+                      colormap=None,
+                      subplot_shape=None,
+                      figsize=None,
+                      **kwargs
+                      ):
+    """Visualization for metrics with statistical error bounds.
+
+    Plots a given metric and its given error (as described by the `error_bars` or `conf_intervals`)
+
+    This function takes in a :class:`fairlearn.metrics.MetricFrame` with precomputed metrics
+    and metric errors and a `error_bars` or `conf_intervals` array to interpret the columns
+    of the :class:`MetricFrame`.
+
+    The items at each index of the given `metrics` array and given `error_bars` or `conf_intervals`
+    array should correspond to a pair of the same metric and metric error, respectively.
+
+    Note:
+        If given many metrics to plot, this function plots them in only 1 row
+        with the number of subplot columns matching the number of metrics
+
+    Parameters
+    ----------
+    metric_frame : fairlearn.metrics.MetricFrame
+        The collection of disaggregated metric values, along with the metric errors.
+
+    kind : str, optional
+        The type of plot to display. i.e. "bar", "line", etc.
+        List of options is detailed in :meth:`pandas.DataFrame.plot`
+
+    metrics : str or list of str
+        The name of the metrics to plot. Should match columns from the given :class:`MetricFrame`.
+
+    error_bars : str or list of str
+        The name of the error metrics to plot.
+        Should match columns from the given :class:`MetricFrame`.
+        Error bars quantify the bounds relative to the base metric.
+
+        Example:
+            If the error bar for a certain column is :code:`[0.01, 0.02]`
+            and the metric is 0.6
+            Then the plotted bounds will be :code:`[0.59, 0.62]`
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper errors. i.e. :code:`[0.01, 0.02]`
+
+    conf_intervals : str or list of str
+        The name of the confidence intervals to plot.
+        Should match columns from the given :class:`MetricFrame`.
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper bounds. i.e. :code:`[0.59, 0.62]`
+
+    axs : :class:`matplotlib.axes._axes.Axes`\
```

Comment:
Make sure that these render properly. Above you didn't use ReST directives, and here you're using them. Only one way works, although I keep forgetting which one... anyway, make sure to run the doc build and check every line